docs/src/main/java/io/quarkus/docs/generation/YamlMetadataGenerator.java

```diff
@@ -179,18 +173,23 @@ public static void main(String[] args) throws Exception {
             }
         }
 
-        ObjectMapper om = new ObjectMapper(new YAMLFactory().enable(YAMLGenerator.Feature.MINIMIZE_QUOTES));
-        Map<String, DocMetadata> metadata = index.metadataByFile();
-        Map<String, Collection<String>> errorsByFile = errors.errorsByFile(metadata);
-
-        om.writeValue(targetDir.resolve("indexByType.yaml").toFile(), index);
-        om.writeValue(targetDir.resolve("indexByFile.yaml").toFile(), metadata);
+        return index;
+    }
 
-        om.writeValue(targetDir.resolve("errorsByType.yaml").toFile(), errors);
-        om.writeValue(targetDir.resolve("errorsByFile.yaml").toFile(), errorsByFile);
+    boolean includeFile(String fileName) {
+        if (fileName.equals("attributes.adoc") || fileName.startsWith("README.adoc")) {
```

Comment:
Probably a good idea to add `attributes-local.adoc` here as it will come soon.

For my `unlinkedguides.java` script, I have all these exceptions (suffixed with `.adoc` when checking):
```
        "0-glossary",
        "platform-include",
        "quarkus-intro",
        "duration-format-note",
        "status-include",
        "quarkus-blaze-persistence",
        "attributes",
       "attributes-local",
        "smallrye-kafka-incoming",
        "smallrye-kafka-outgoing",
        "faq",
        "building-substrate-howto",
        "README",
        "gradle-config",
        "amazon-credentials",
        "kogito-dev-services-build-time-config"
```

Reply:
Note that maybe, we should move at least some of them to an include directory to avoid having them at the root. That would prevent us from maintaining this list.